### ISSUE TITLE
Saalfield Repo Not found

### ISSUE THREAD

**Issue by ilan-gold** [glencoesoftware/bioformats2raw#44]
Hello,

I am noticing some behavior in my travis build of `raw2ometiff` (which references the current snapshot here, `0.2.3`):

```
...
   > Could not find org.janelia.saalfeldlab:n5-zarr:0.0.4.

     Searched in the following locations:

       - https://jcenter.bintray.com/org/janelia/saalfeldlab/n5-zarr/0.0.4/n5-zarr-0.0.4.pom

       - https://repo.maven.apache.org/maven2/org/janelia/saalfeldlab/n5-zarr/0.0.4/n5-zarr-0.0.4.pom

       - https://artifacts.openmicroscopy.org/artifactory/repo/org/janelia/saalfeldlab/n5-zarr/0.0.4/n5-zarr-0.0.4.pom

       - https://repo.glencoesoftware.com/repository/n5-zarr-snapshots/org/janelia/saalfeldlab/n5-zarr/0.0.4/n5-zarr-0.0.4.pom

       - https://repo.glencoesoftware.com/repository/bioformats2raw2ometiff/org/janelia/saalfeldlab/n5-zarr/0.0.4/n5-zarr-0.0.4.pom

       - https://saalfeldlab.github.io/maven/org/janelia/saalfeldlab/n5-zarr/0.0.4/n5-zarr-0.0.4.pom

     Required by:

         project : > com.glencoesoftware:bioformats2raw:0.2.3-SNAPSHOT:20200623.150117-4
 ...
```

The full logs are here: https://travis-ci.com/github/hubmapconsortium/portal-containers/builds/172705798

I think it is related to #41 - when I go to the github.io address of the maven repo in the error, I get a 404.

**Comment by melissalinkert** [comment]
https://github.com/glencoesoftware/raw2ometiff/pull/36 brings the raw2ometiff dependencies back into sync with #41, so I'd expect that to fix this problem.

**Comment by ilan-gold** [comment]
@melissalinkert Does this mean that the `raw2ometiff` `0.2.4` tag is not going to work anymore?  I am currently using bioformats2raw `0.2.1` -> `raw2ometiff` `0.2.4` but I would be happy to update.  Will there be a new release?  I used this combination because I was under the impression that there was a compatibility issue between other versions, but if this is resolving that, that would be great.

**Comment by chris-allan** [comment]
@ilan-gold: Yes, the 0.2.4 tag will not longer build with https://saalfeldlab.github.io/maven/ gone. You will need to upgrade to `0.2.4` of `bioformats2raw` and `0.2.6` of `raw2ometiff`.

**Comment by axtimwalde** [comment]
https://saalfeldlab.github.io/maven/ contains all artifacts from before the transfer to maven.scijava.org.  The repository is also not gone.  May be a hickup on github.io?  Or do you try to fetch newer artifacts than what's n5-0.0.4 was never released on the saalfeldlab maven repository, please add the maven.scijava.org repository to your pom file when you bumpt to n5-zarr-0.0.4.

**Comment by chris-allan** [comment]
Thanks for commenting @axtimwalde . Apologies for wasting people's time.

I think the source of the confusion here is me going too fast and not thoroughly reading all the commits. I made a poor assumption from reading the commit message that 388a19adc676f100b609d656c22ffd3eb737f8ec removed the JARs from `https://saalfeldlab.github.io/maven/`. I'm sure it didn't help at the time that `https://saalfeldlab.github.io/maven/org/janelia/saalfeldlab/n5-zarr/` for example returns 404.

The actual core software issue is that for testing `raw2ometiff` was consuming `SNAPSHOT` versions of `bioformats2raw` whose dependencies were updated to n5-zarr 0.0.4. `raw2ometiff` itself was still on 0.0.3-SNAPSHOT versions we built and deployed ourselves which included saalfeldlab/n5-zarr#2 and of course consequently has not had the`maven.scijava.org` change. Thus, we have dependencies being included without the requisite repositories being on the search path.

Versions (`0.2.4` of `bioformats2raw` and `0.2.6` of `raw2ometiff`) have been released with all the requisite changes made and depend on n5-zarr 0.0.4.

/cc @ctrueden
